jupyterhub/templates/proxy/deployment.yaml

```diff
@@ -100,6 +100,9 @@ spec:
             {{- if .Values.debug.enabled }}
             - --log-level=debug
             {{- end }}
+            {{- if .Values.hub.config.subdomain_host }}
```

Comment:
```suggestion
            {{- if .Values.hub.config.JupyterHub.subdomain_host }}
```

Reply:
This can error if we dont always have JupyterHub defined as a dict etc, maybe we can assume we have? I dont remember.

Reply:
It's already defined 😄 
https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/92ef7265bbd1c03b3fbef9be82c2f7f38518b8df/jupyterhub/values.yaml#L35-L38